python/device/iothub_client_python/CMakeLists.txt

```diff
@@ -48,7 +47,17 @@ endif()
 
 
 # get python
-find_package(PythonLibs 2.7 REQUIRED)
+if (${python_3})
+    set(Python_ADDITIONAL_VERSIONS 3)
+    add_definitions(-DPY_MARJOR_VERSION=3)
```

Comment:
I found the PY_MAJOR_VERSION check from the python migration docs, but didn't seem to be in effect when I built without this additional add_definitions line. Not sure if it's something else in our boost setup or otherwise.